### PR TITLE
Updating example run instructions in README.md

### PR DESCRIPTION
**Pull request by ryanmorrison22**
I have updated the README file example using clusters-small.fasta to reflect the most recent changes to KPop.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,86 +98,98 @@ Download file `clusters-small.fasta` from the directory `test`. Then run the fol
 ```bash
 export K=5
 date
-for CLASS in C1 C2 C3 C4 C5 C6 C7 C8 C9 C10; do cat clusters-small.fasta | awk -v CLASS=$CLASS '{nr=(NR-1)%4; ok=(nr==0?$0~("-"CLASS"$"):nr==1&&ok); if (ok) print}' | KPopCount -k $K -l $CLASS -f /dev/stdin | KPopCountDB -k /dev/stdin -R "~." -A $CLASS -L $CLASS -N -D --table-transform none -t /dev/stdout; done | KPopCountDB -k /dev/stdin -o Classes.$K -v
+for CLASS in C1 C2 C3 C4 C5 C6 C7 C8 C9 C10; do cat clusters-small.fasta | awk -v CLASS=$CLASS '{nr=(NR-1)%4; ok=(nr==0?$0~("-"CLASS"$"):nr==1&&ok); if (ok) print}' | KPopCount -k $K -L -f /dev/stdin | KPopCountDB -k /dev/stdin -R "~." -A $CLASS -L $CLASS -N -D --table-transform none -t /dev/stdout; done | KPopCountDB -k /dev/stdin -o Classes -v
```
Comment (PaoloRibeca) [inline README.md]: OK to remove -l with -L. Better to leave the .$K though, if anyone wanted to try different values?

File: README.md [hunk 2]
```diff
@@ -98,86 +98,98 @@ Download file `clusters-small.fasta` from the directory `test`. Then run the fol
...
 KPopTwist -i Classes -v
-cat clusters-small.fasta | awk -v K="$K" '{nr=(NR-1)%4; if (nr==2) split($0,s,"[>-]"); if (nr==3) print ">"s[2]"-"s[3]"\n"$0}' | KPopCount -k $K -L -f /dev/stdin | KPopTwistDB -i T Classes.$K -k /dev/stdin -o t /dev/stdout | KPopTwistDB -i T Classes.$K -i t Classes.$K -s /dev/stdin Classes.$K -v
-echo -n ">>> Misclassified sequences: "; cat Classes.KPopSummary.txt | awk -F '\t' 'BEGIN{OFS="\t"} {$1=gensub("-","\"\t\"",1,$1); print}' | awk -F '\t' '{if ($2!=$7) print}' | wc -l
+cat clusters-small.fasta | awk -v K="$K" '{nr=(NR-1)%4; if (nr==2) split($0,s,"[>-]"); if (nr==3) print ">"s[2]"-"s[3]"\n"$0}' | KPopCount -k $K -L -f /dev/stdin | KPopTwistDB -i T Classes -k /dev/stdin -o t /dev/stdout | KPopTwistDB -i T Classes -i t Classes -s /dev/stdin Class_prediction -v
+echo -n ">>> Misclassified sequences: "; cat Class_prediction.KPopSummary.txt | awk -F '\t' 'BEGIN{OFS="\t"} {$1=gensub("-","\"\t\"",1,$1); print}' | awk -F '\t' '{if ($2!=$7) print}' | wc -l
```
Comment (PaoloRibeca) [inline README.md]: I like Class_prediction, but perhaps we can call it Class_prediction.$K ?

File: README.md [hunk 3]
```diff
@@ -98,86 +98,98 @@ Download file `clusters-small.fasta` from the directory `test`. Then run the fol
...
 That should produce an output such as this one:
 
 ```
-Wed 24 Jan 11:18:16 BST 2024
-This is KPopCountDB version 37 [22-Jan-2024]
- compiled against: BiOCamLib version 242 [23-Jan-2024];
-                   KPop version 366 [23-Jan-2024]
- (c) 2020-2024 Paolo Ribeca <paolo.ribeca@gmail.com>
+Wed 14 Feb 16:46:55 GMT 2024
+╔━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╗
```
Comment (PaoloRibeca) [inline README.md]: I would remove the fancy formatting, as it makes things more readable :) . I can do that later if you want

File: README.md [hunk 4]
```diff
@@ -98,86 +98,98 @@ Download file `clusters-small.fasta` from the directory `test`. Then run the fol
...
    ```
    runs `KPopCount` (with *k*=5) on each training sequence belonging to CLASS; the results, which are text files each one containing a list of *k*-mers with their respective frequencies, are concatenated and sent to `KPopCountDB` through a pipe &mdash; if you wanted to look into the format, you could do so with the command
    ```bash
-   $ cat clusters-small.fasta | awk -v CLASS="$CLASS" '{nr=(NR-1)%4; ok=(nr==0?$0~("-"CLASS"$"):nr==1&&ok); if (ok) print}' | KPopCount -l $CLASS -f /dev/stdin -k "$K" | less
+   $ cat clusters-small.fasta | awk -v CLASS=$CLASS '{nr=(NR-1)%4; ok=(nr==0?$0~("-"CLASS"$"):nr==1&&ok); if (ok) print}' | KPopCount -k $K -L -f /dev/stdin | less
    ```
-   After that, `KPopCountDB` collects all the spectra for each class into a temporary database, replaces them with an average of the inputs, names the averaged spectrum `$CLASS`, and re-outputs it in the same format used before for the spectra produced by `KPopCount`
+   After that, `KPopCountDB` collects all the spectra for each class into a temporary database, names the spectrum `$CLASS`, and re-outputs it in the same format used before for the spectra produced by `KPopCount`
```
Comment (PaoloRibeca) [inline README.md]: I think we should clarify that the new spectrum is a combination of the spectra of the sequences in the class.

Perhaps `into a temporary database, replaces them with a combination of the input spectra, names the combined spectrum`?

File: README.md [hunk 5]
```diff
@@ -98,86 +98,98 @@ Download file `clusters-small.fasta` from the directory `test`. Then run the fol
...
 3. The command
    ```bash
-   $ KPopCountDB -f /dev/stdin -o Classes -v
+   $ KPopCountDB -k /dev/stdin -o Classes -v
    ```
-   receives the 10 averaged spectra, one per class, and outputs them to a binary database called `Classes` (actually that corresponds to a file, which gets automatically named `Classes.KPopCounter`)
+   receives the 10 spectra, one per class, and outputs them to a binary database called `Classes` (actually that corresponds to a file, which gets automatically named `Classes.KPopCounter`)
```
Comment (PaoloRibeca) [inline README.md]: OK

**Review by PaoloRibeca — CHANGES_REQUESTED** [review]
I would perhaps leave the .$K in, but apart from that all good 👍

File: README.md (hunk 1 above)
Comment (ryanmorrison22) [reply README.md]: Sure, I have made the changes

File: README.md (hunk 2 above)
Comment (ryanmorrison22) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (ryanmorrison22) [reply README.md]: I have added that sentence.

File: README.md (hunk 3 above)
Comment (ryanmorrison22) [reply README.md]: Done.

**Pull request merged**